packages/create-neutrino-project/commands/init/templates/test/eslintrc.js

```diff
@@ -0,0 +1,5 @@
+const { Neutrino } = require('neutrino');
```

Comment:
We can probably move this file out of the `test` directory and into a more appropriate one.